index.js

```diff
@@ -9,4 +9,45 @@ function closeMenu() {
 }
 
 openMenu();
-closeMenu();
\ No newline at end of file
+closeMenu();
+
+const modalBtn = document.querySelector('.modal-btn');
+
+const data = document.querySelector('.modal');
+
+const dataValue = document.createElement('div');
+
+dataValue.classList.add('pop-up');
+
+dataValue.innerHTML = `
+<h3 class="pop-up-heading">Multi Post Story</h3>
+<i class="fa fa-times-circle pop-up-close"></i>
+<ul class="pop-up-skills">
+  <li class="pop-up-skls">HTML</li>
+  <li class="pop-up-skls">CSS</li>
+  <li class="pop-up-skls">Ruby on Rails</li>
+</ul>
+<div class="pop-up-container">
+<img src="Assets/Snapshoot Portfolio.png" class="pop-up-background" alt="Pop-up Image">
+<div class="pop-up-right">
+<p class="pop-up-text">Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+  Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+   when an unknown printer took a galley of type and scrambled it to make a type 
+   specimen book. It has survived not only five centuries, but also the leap into 
+   electronic typesetting, remaining essent</p>
+  <div class="pop-up-button">
+    <a href="#" class="pop-up-btn">Livelink<i class="fa fa-arrow-circle-down"></i></a>
+    <a href="#" class="pop-up-btn">Sourcelink<i class="fa fa-github"></i></a>
+    </div>
+    </div>
+    </div>`;
+data.appendChild(dataValue);
+
+const popClose = document.querySelector('.pop-up-close');
+popClose.addEventListener('click', () => {
+  dataValue.classList.add('displayModal');
+});
+
+modalBtn.addEventListener('click', () => {
+  dataValue.classList.remove('displayModal');
+});
```

Comment:
- [ ] In order to have the popup for each of the projects with their dynamic contents, you need to implement your JavaScript code in such a way that there will be an array of objects to store JavaScript objects containing the information for each project.

- [ ] Each object contains at least the following information about the project: name, description, featured image, technologies, link to live version, and link to the project source code. i.e:
```JS
const projects = [
  {
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
  }
  ...
]
```

- [ ] You can then create the projects section dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in your existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.

- [ ] Modal popup data is then populated with the data from the projects array. Following this process will make it easy to add projects in the future. It means that you will only add it in one place - as another object in the projects array.